core/scripts/create_adapter_plugins.py

```diff
@@ -1,323 +1,5 @@
-#!/usr/bin/env python
-import argparse
-import sys
-from pathlib import Path
-
-PROJECT_TEMPLATE = """
-name: dbt_{adapter}
-version: {version}
-config-version: 2
-
-macro-paths: ["macros"]
-"""
-
-
-SETUP_PY_TEMPLATE = '''
-#!/usr/bin/env python
-from setuptools import find_namespace_packages, setup
-
-package_name = "dbt-{adapter}"
-# make sure this always matches dbt/adapters/{adapter}/__version__.py
-package_version = "{version}"
-description = """The {adapter} adapter plugin for dbt"""
-
-setup(
-    name=package_name,
-    version=package_version,
-    description=description,
-    long_description=description,
-    author={author_name},
-    author_email={author_email},
-    url={url},
-    packages=find_namespace_packages(include=['dbt', 'dbt.*']),
-    include_package_data=True,
-    install_requires=[
-        "{dbt_core_str}",{dependencies}
-    ]
+# The create_adapter_plugins script is being replaced by a new interactive cookiecutter scaffold
+# that can be found https://github.com/dbt-labs/dbt-database-adapter-scaffold
+print(
+    "To create a new adapter please visit https://github.com/dbt-labs/dbt-database-adapter-scaffold"
```

Comment:
Maybe add "This script has been deprecated. To create a new adapter..."

Reply:
Will do 👍